packages/ui-library/src/components/background-paper/index.tsx

```diff
@@ -12,7 +12,7 @@ const StyledBackgroundPaper = styled(Paper)`
   }) => `
     background-color: ${colors[mode].background.quartery};
     border-radius: ${spacing(4)};
-    padding: ${spacing(8)};
+    padding: ${spacing(10)} ${spacing(8)};
```

Comment:
this doesnt break anything?

Reply:
The designer has changed the paddings and margins of some papers and modals in the UI. I created a task for this (BLY-1668), but I needed this change here to have the component finished with this task